modules/tensor_mechanics/doc/content/documentation/modules/tensor_mechanics/generalized_plane_strain.md

```diff
@@ -0,0 +1,125 @@
+
+## Formulation
+
+For the description purpose, it is convenient to introduce the coordinate system so that the $x$-$y$ plane is positioned transverse to the infinite out-of-plane direction and the $z$-axis runs in the out-of-plane direction. Under the simplest generalized plane strain conditions, the in-plane stresses are not functions of $z$, implying that the in-plane strains are not functions of $z$ from the constitutive relationship for linearly elastic and homogeneous materials. The generalized plane strain problem is 2D in presentation, defined in a 2D domain.
+
+The kinematical equations of the generalized plane strain problem are identical to those for the plane stress or strain problems, given as
+\begin{equation}
+\left\{\begin{matrix}
+\epsilon_{xx} \\
+\epsilon_{yy} \\
+\gamma_{xy}
+\end{matrix}\right\}
+=
+\left\{\begin{matrix}
+\frac{\partial u}{\partial x} \\
+\frac{\partial v}{\partial y} \\
+\frac{\partial v}{\partial x} + \frac{\partial u}{\partial y}
+\end{matrix}\right\}
+\end{equation}
+
+The equilibrium equations for the generalized plane strain problem in the $x$-$y$ plane are given as
+\begin{equation}
+\begin{matrix}
+\frac{\partial \sigma_{xx}}{\partial x} + \frac{\partial \sigma_{xy}}{\partial y} + \bar{f}_{x} = 0 \\
+\frac{\partial \sigma_{xy}}{\partial x} + \frac{\partial \sigma_{yy}}{\partial y} + \bar{f}_{y} = 0
+\end{matrix}
+\end{equation}
+in $A$ where $\left\{ \bar{f} \right\}$ are the body forces.
+
+The constitutive equations in terms of stress-strain relationship are given as
+\begin{equation}
+\left\{ \begin{matrix}
+\sigma_{xx} \\
+\sigma_{yy} \\
+\sigma_{zz} \\
+\tau_{xy}
+\end{matrix} \right\}
+=
+\left[ \begin{matrix}
+C_{11} & C_{12} & C_{13} & 0 \\
+C_{12} & C_{22} & C_{23} & 0 \\
+C_{13} & C_{23} & C_{33} & 0 \\
+0 & 0 & 0 & C_{66}
+\end{matrix} \right]
+\left( \left\{ \begin{matrix}
+\epsilon_{xx} \\
+\epsilon_{yy} \\
+\epsilon_{zz} \\
+\gamma_{xy}
+\end{matrix} \right\}
+-
+\left\{ \begin{matrix}
+\alpha_{x} \\
+\alpha_{y} \\
+\alpha_{z} \\
+0
+\end{matrix} \right\} \Delta T \right)
+\end{equation}
+where $C_{ij}$ are material's stiffness coefficients using Voigt notation, $\alpha_{x}$, $\alpha_{y}$ and $\alpha_{z}$ are the material's thermal expansion coefficients and $\Delta T$ is the change in temperature.
+
+A further condition is required associated with the out-of-plane direction. If a constant strain is prescribed as the deformation compatibility condition in the $z$-direction
```

Comment:
Add a third level (`###`) header here, like `### Prescribed Out-of-Plane Strain`